client/blocks/reader-post-card/style.scss

```diff
@@ -402,7 +402,7 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 	}
 
 	@include breakpoint( ">480px" ) {
```

Comment:
can we remove this whole selector now since its the same as https://github.com/Automattic/wp-calypso/pull/12217/files#diff-ead3a37b620641e62b8bd6d1639fefdbL396 ?